robottelo/cli/gpgkey.py

```diff
@@ -38,25 +38,23 @@ def info(cls, options=None):
         cls.command_sub = 'info'
 
         result = cls.execute(
-            cls._construct_command(options), output_format='csv')
+            cls._construct_command(options), output_format='json')
+
+        if isinstance(result, dict) or len(result) == 0:
```

Comment:
`result` should always be a dict when using json output, I think this method can be as simple as:

```python
def info(cls, options=None):
    """Get GPG Key information."""
    cls.command_sub = 'info'
    return cls.execute(
        cls._construct_command(options, output_format='json')
```